app/javascript/components/city-name.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import { WhiteTitleLarge } from '@components/typography';
+import { LoadingMessage, ErrorMessage } from '@messages/default-messages';
+
+const getCityName = gql`
+  {
+    primaryLocation {
```

Comment:
How is this being set? (ie how would I be able to change it to Boulder)

Reply:
In the .env file, you can change the value set for `PRIMARY_CITY_NAME` based on the available cities in the data base. graphQL queries the database to get the set city name.

Reply:
Will I be able to display two cities at the same time? Can I set two different values for PRIMARY_CITY_NAME in the .env file if I want to do so?

Reply:
No this only works for one city.

Reply:
...so far :wink: